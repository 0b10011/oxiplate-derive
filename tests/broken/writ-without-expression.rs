use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{{ ` }}"]
struct Data {}

fn main() {
    print!("{}", Data {});
}
