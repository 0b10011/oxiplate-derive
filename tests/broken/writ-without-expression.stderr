error: Expected an expression.
 --> tests/broken/writ-without-expression.rs:4:25
  |
4 | #[oxiplate_inline = "{{ ` }}"]
  |                         ^
